po/tr.po

```diff
@@ -0,0 +1,302 @@
+# Turkish translation for Comic Sticks (xkcd-gtk).
+#
+# Copyright (C) 2015-2022 Ryan Koesters
+# This file is distributed under the same license as the com.github.rkoesters.xkcd-gtk package.
+#
+# Sabri Ünal <libreajans@gmail.com>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.rkoesters.xkcd-gtk\n"
+"Report-Msgid-Bugs-To: https://github.com/rkoesters/xkcd-gtk/issues\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2022-06-21 20:56+0300\n"
+"Last-Translator: Sabri Ünal <libreajans@gmail.com>\n"
+"Language-Team: Turkish <gnome-turk@gnome.org>\n"
+"Language: tr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"X-Generator: Poedit 3.1\n"
+"X-Project-Style: gnome\n"
+
+#: internal/widget/about-dialog.go:17
+msgid "A simple xkcd viewer written in Go using GTK+3"
+msgstr "Go dili ve GTK+3 kullanılarak yazılmış basit xkcd görüntüleyici"
+
+#: internal/widget/window-menu.go:135 internal/widget/app-menu.ui:38
+msgid "About Comic Sticks"
+msgstr "Karikatürler Hakkında"
+
+#: internal/widget/window-menu.go:123 internal/widget/app-menu.ui:28
+msgid "About XKCD"
+msgstr "XKCD Hakkında"
+
+#: internal/widget/bookmarks-menu.go:71 internal/widget/context-menu.go:23
+msgid "Add to bookmarks"
+msgstr "Yer imlerine ekle"
+
+#: internal/widget/properties-dialog.go:102
+msgid "Alt text"
+msgstr "Alternatif metin"
+
+#: internal/widget/shortcuts-window.ui:118
+msgid "Application actions"
+msgstr "Uygulama eylemleri"
+
+#: internal/widget/bookmarks-menu.go:53 internal/widget/shortcuts-window.ui:92
+msgid "Bookmarks"
+msgstr "Yer imleri"
+
+#: data/com.github.rkoesters.xkcd-gtk.appdata.xml.in:17
+msgid "Cache comics for later offline viewing"
+msgstr "Karikatürleri daha sonra çevrimdışı görüntülemek için önbelleğe al"
+
+#: internal/widget/application-window.go:273
+msgid "Checking for new comic..."
+msgstr "Yeni karikatür denetleniyor..."
+
+#: internal/widget/application.go:17
+#: data/com.github.rkoesters.xkcd-gtk.desktop.in:5
+#: data/com.github.rkoesters.xkcd-gtk.appdata.xml.in:9
+msgid "Comic Sticks"
+msgstr "Karikatürler"
```

Comment:
As the name of the app, "Comic Sticks" probably should not be translated to "Cartoons" (I don't speak this language, I'm going off of what Google Translate tells me this translates to). Looking at what previous translators have done:

3 of the 4 existing languages leave the app name unchanged:

* https://github.com/rkoesters/xkcd-gtk/blob/main/po/fr.po#L60
* https://github.com/rkoesters/xkcd-gtk/blob/main/po/it.po#L60
* https://github.com/rkoesters/xkcd-gtk/blob/main/po/nl.po#L61

The Portuguese translation does translate the app name to "Comic Book Puppets" which seems like a reasonable attempt to translate the app name (but, again, English is the only language in which I am proficient):

* https://github.com/rkoesters/xkcd-gtk/blob/main/po/pt.po#L63

Given this current inconsistency, it seems like I should make a decision on how this app should handle translations of its own name. While the words of the app name are translatable, the app name being a pun on "comic strips"/"stick figures" means that aspect of the name will likely be lost in translation. My first thought is to encourage translating the app name if some of the punnyness can be translated as well, otherwise leave the name as-is. What are your thoughts on that?

Reply:
The pun in "Comic Sticks" (being a mash-up of "comic strip" and "stick figures") is a big part of the app's name to me. Since the pun doesn't translate well, then let's just not translate it. I'll update the app to not localize that string. Let's go ahead and merge this as-is, as the Makefile will help clean up references to the deleted localization string.

Thank you for all the information, and thank you again for the pull request!